### PR TITLE
Allow query parameters to be passed with query

### PR DESCRIPTION
**Pull request by kislyuk**

### CODE REVIEW

File: dcpquery-api.yml [hunk 1]
```diff
@@ -146,6 +146,12 @@ components:
         query:
           type: string
           description: Query to run, given as a SQL string.
+        params:
+          type: object
+          description: >
```
Comment (MDunitz) [inline dcpquery-api.yml]: Can you include an example that shows how this helps avoid the escaping issue

File: tests/integration/test_endpoints.py [hunk 2]
```diff
@@ -31,6 +31,14 @@ def test_query(self):
         self.assertEqual(response.json['query'], query)
         self.assertGreaterEqual(response.json['results'][0]['count'], 0)
 
+        query = "SELECT count(*) FROM FILES WHERE SIZE > :s;"
```
Comment (MDunitz) [inline tests/integration/test_endpoints.py]: would it be worth it to add a test showing this gives the same result as running the query without params 
`"SELECT count(*) FROM FILES WHERE SIZE > 0;"`
Also would it be worth it to return the query that was actually run with the params in place to help debug issues?

Comment (kislyuk) [reply tests/integration/test_endpoints.py]: 1. Yes, but if the database undergoes a change in the number of files as we run the test, the result will change. So we need some way to ensure atomicity in the test. I tried passing multiple queries (`select count(*) from files; select count(*) from files where size > 0;` but the first result gets shadowed by the second. Do you know how to ensure atomicity?

2. Yes, that's a great idea but I'd like to tackle it separately. Filed #186 to track that.

File: dcpquery-api.yml (hunk 1 above)
Comment (kislyuk) [reply dcpquery-api.yml]: Yes, absolutely, but I would like to merge https://github.com/HumanCellAtlas/dcp-cli/pull/359 first, and make a complete suite of documentation including the use of this.

Comment (kislyuk) [reply dcpquery-api.yml]: Filed https://github.com/HumanCellAtlas/query-service/issues/187 to track this.

File: tests/integration/test_endpoints.py (hunk 2 above)
Comment (MDunitz) [reply tests/integration/test_endpoints.py]: Maybe if you reset the db connection between queries?

**Pull request merged**